doc/user-guide/04-scenarios.adoc

```diff
@@ -1331,3 +1331,109 @@ Finished recovering your system. You can explore it under '/mnt/local'.
 
 Finished in 361 seconds.
 ----
+
+== Using ReaR to repair your system
+Instead of using your ReaR image to completely recover your system from bare
+metal (as illustrated in most of the above scenarios), you can also use it
+as a live media to boot a broken but hopefully repairable system.
+
+Once booted on your recovery image, the `repair` workflow will:
+
+* activate all Volume Groups
+
+* offer to decrypt any LUKS-encrypted filesystem that may be present
+
+* mount all the target filesystems (including the most important virtual 
+  ones) below `/mnt/local`
+
+thereby making it possible for you to explore your system at will,
+correcting any configuration mistake that may have prevented its startup, 
+or allowing you to simply `chroot` into it and further repair it using its 
+own administrative tools.
+
+One important point to remember is that the `repair` workflow on its own 
+won't modify the target system in any way. Of course, once the target 
+filesystems are mounted you, as the administrator, may decide to do so
+manually.
+
+**Beware:** The `repair` workflow can only be used on the system where
+the rescue image was generated, as it bases its logic on the filesystem
+layout description file generated during the run of the `mkrescue` or
+`mkbackup` workflows.
+
+Here are the steps you would typically follow:
+
+=== Create your recovery image
+Using any of the techniques described in the other scenarios, create a 
+ReaR recovery image for your system (through `rear mkrescue` or `rear 
+mkbackup`). If you only take the `repair` workflow into consideration, it 
+doesn't matter whether you also make a backup of your system or not 
+(obviously, you'd better cover all your bases and make sure you'd be able to 
+perform a full `recover` as well should the need occur).
+
+Please note, that by default ReaR only includes in the recovery image the 
+tools it will need to recover the system. If you anticipate the need for 
+some extra tools in the context of a repair operation (e.g. tools that you 
+might need in the event `chroot`ing into the target system doesn't work), you 
+should make sure to include them in your recovery image by adding to the 
+`PROGS` or `REQUIRED_PROGS` configuration variables (please refer to the 
+comments in `default.conf` for the exact meaning of each).
+
+=== Booting on the recovery image
+Arrange for the target system to boot on your recovery image as you would in
+any of the other scenarios.
+
+=== Launching the repair workflow
+Issue the `rear repair` command to launch the workflow:
+
+----
+RESCUE pc-pan:~ # rear -v repair
```

Comment:
I think you should add the repair workflow to those
that are always set to be verbose in usr/sbin/rear
```
# The following workflows are always verbose:
case "$WORKFLOW" in
    (validate|shell|recover)
        VERBOSE=1
        ;;
esac
```

Alternatively replace `LogPrint` where needed with `LogUserOutput`
or `LogPrintError` when you need to show messages to the user
in any case, cf.
https://github.com/rear/rear/blob/master/usr/share/rear/lib/_input-output-functions.sh#L433

Reply:
Changes committed.